diff --git tools/reformat.sh tools/reformat.sh
index bdb856d..e49855f 100755
--- tools/reformat.sh
+++ tools/reformat.sh
@@ -43,9 +43,12 @@ done
 #$go get -d -u ./... || true
 $go mod tidy || true
 
+# Generate ent database files
+$go generate
+
 # Gazelle
-find . -name '*.pb.go' -delete
-rm -f $(find . -name '*.proto' | sed -e 's/[^/]*$/BUILD.bazel/')
+find . -name '*.pb.go' -not -path './pkg/proto/bazelbuild/*' -delete
+rm -f $(find . -name '*.proto' -not -path './pkg/proto/bazelbuild/*' -not -path './third_party/*' | sed -e 's/[^/]*$/BUILD.bazel/')
 $gazelle
 
 # bzlmod
@@ -55,13 +58,14 @@ bazel mod tidy
 $gofumpt -w -extra "$(pwd)"
 
 # Protobuf
-find . -name '*.proto' -exec "$clang_format" -i {} +
+find . -name '*.proto' -not -path './frontend/*' -exec "$clang_format" -i {} +
 
 # Generated .pb.go files
 find bazel-bin/ -path "*${go_module_name}*" -name '*.pb.go' -delete || true
 bazel build --output_groups=go_generated_srcs $(bazel query --output=label 'kind("go_proto_library", //...)')
+third_party/bazel/download_protofiles.sh
 find bazel-bin/ -path "*${go_module_name}*" -name '*.pb.go' | while read f; do
-  cat $f > $(echo $f | sed -e "s|.*/${go_module_name}/||")
+  cat $f > $(echo $f | sed -e 's|^bazel-bin/||' -e 's|/[^/]*_go_proto_/.*/|/|')
 done
 
 # Files embedded into Go binaries
