diff --git tools/github_workflows/workflows_template.libsonnet tools/github_workflows/workflows_template.libsonnet
index 73bc06c..916c177 100644
--- tools/github_workflows/workflows_template.libsonnet
+++ tools/github_workflows/workflows_template.libsonnet
@@ -4,15 +4,6 @@
       name: 'linux_amd64',
       extension: '',
     },
-    {
-      name: 'linux_386',
-      extension: '',
-      testPlatform: 'linux_amd64',
-    },
-    {
-      name: 'linux_arm',
-      extension: '',
-    },
     {
       name: 'linux_arm64',
       extension: '',
@@ -47,13 +38,6 @@
               platform_name: 'linux_amd64',
               upload: true,
             },
-            {
-              bazel_os: 'windows',
-              cross_compile: false,
-              os: 'windows-latest',
-              platform_name: 'windows_amd64',
-              upload: false,
-            },
           ],
         },
       },
@@ -173,12 +157,12 @@
     'main.yaml': {
       name: 'main',
       on: { push: { branches: ['main'] } },
-      jobs: getJobs(binaries, containers, setupSteps, extraSteps, true, false),
+      jobs: getJobs(binaries, containers, setupSteps, extraSteps, true, true),
     },
     'pull-requests.yaml': {
       name: 'pull-requests',
       on: { pull_request: { branches: ['main'] } },
-      jobs: getJobs(binaries, containers, setupSteps, extraSteps, false, false),
+      jobs: getJobs(binaries, containers, setupSteps, extraSteps, false, true),
     },
   },
 }
