diff --git tools/github_workflows/workflows_template.libsonnet tools/github_workflows/workflows_template.libsonnet
index 89861ab..20855a7 100644
--- tools/github_workflows/workflows_template.libsonnet
+++ tools/github_workflows/workflows_template.libsonnet
@@ -6,18 +6,6 @@
       buildJustBinaries: false,
       extension: '',
     },
-    {
-      name: 'linux_386',
-      buildAndTestCommand: 'test --test_output=errors',
-      buildJustBinaries: false,
-      extension: '',
-    },
-    {
-      name: 'linux_arm',
-      buildAndTestCommand: 'build',
-      buildJustBinaries: false,
-      extension: '',
-    },
     {
       name: 'linux_arm64',
       buildAndTestCommand: 'build',
@@ -72,9 +60,13 @@
           name: 'Bazel mod tidy',
           run: 'bazel mod tidy',
         },
+        {
+          name: 'Go generate',
+          run: 'go generate',
+        },
         {
           name: 'Gazelle',
-          run: "rm -f $(find . -name '*.pb.go' | sed -e 's/[^/]*$/BUILD.bazel/') && bazel run //:gazelle",
+          run: "rm -f $(find . -name '*.pb.go' -not -path './pkg/proto/bazelbuild/*' | sed -e 's/[^/]*$/BUILD.bazel/') && bazel run //:gazelle",
         },
         {
           name: 'Buildifier',
@@ -86,7 +78,7 @@
         },
         {
           name: 'Clang format',
-          run: "find . -name '*.proto' -exec bazel run @llvm_toolchain_llvm//:bin/clang-format -- -i {} +",
+          run: "find . -name '*.proto' -not -path './frontend/*' -exec bazel run @llvm_toolchain_llvm//:bin/clang-format -- -i {} +",
         },
         {
           name: 'GitHub workflows',
@@ -98,8 +90,9 @@
             if [ -d pkg/proto ]; then
               find . bazel-bin/pkg/proto -name '*.pb.go' -delete || true
               bazel build $(bazel query --output=label 'kind("go_proto_library", //...)')
+              third_party/bazel/download_protofiles.sh
               find bazel-bin/pkg/proto -name '*.pb.go' | while read f; do
-                cat $f > $(echo $f | sed -e 's|.*/pkg/proto/|pkg/proto/|')
+                cat $f > $(echo $f | sed -e 's|^bazel-bin/||' -e 's|/[^/]*_go_proto_/.*/|/|')
               done
             fi
           |||,
@@ -192,12 +185,12 @@
     'master.yaml': {
       name: 'master',
       on: { push: { branches: ['main', 'master'] } },
-      jobs: getJobs(binaries, containers, true, false),
+      jobs: getJobs(binaries, containers, true, true),
     },
     'pull-requests.yaml': {
       name: 'pull-requests',
       on: { pull_request: { branches: ['main', 'master'] } },
-      jobs: getJobs(binaries, containers, false, false),
+      jobs: getJobs(binaries, containers, false, true),
     },
   },
 }
